tests/rptest/services/cluster.py

```diff
@@ -123,8 +123,9 @@ def wrapped(self: HasRedpanda, *args: Any, **kwargs: Any):
 
                     if isinstance(redpanda, RedpandaServiceBase):
                         redpanda.cloud_storage_diagnostics()
-                    if isinstance(redpanda,
-                                  RedpandaServiceCloud | RedpandaServiceCloud):
+                    if isinstance(
+                            redpanda, RedpandaServiceBase
+                            | RedpandaServiceCloud | RedpandaServiceCloud):
```

Comment:
What should be the correct list of classes? `RedpandaServiceBase` doesn't seem to have `raise_on_crash` and `RedpandaServiceCloud` is repeated twice.

Reply:
wow... that's a mistype 2x